staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/BUILD

```diff
@@ -19,6 +19,7 @@ go_library(
     importmap = "k8s.io/kubernetes/vendor/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile",
     importpath = "k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile",
     deps = [
+        "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
```

Comment:
should this be checked in?

Reply:
Yes. You added this dep in passwordfile.go and hack/update-bazel.sh ( part of "Make update") did it for you.